pkg/util/dockerutils.go

```diff
@@ -219,3 +219,30 @@ func GetContainerHealth(container docker.APIContainers) string {
 	}
 	return match
 }
+
+// GetAppContainers retrieves docker containers for a given sitename.
+func GetAppContainers(sitename string) []docker.APIContainers {
+	label := map[string]string{"com.ddev.site-name": sitename}
+	sites, err := FindContainersByLabels(label)
+	if err != nil {
+		log.Fatal("failed to retrieve containers for ", sitename, err)
```

Comment:
It seems like this should probably return an error instead of a fatal? It's only called by Start(), which itself returns an error. Feels like just bubbling this up to the caller is a better approach